internal/forms/forms.go

```diff
@@ -797,9 +795,9 @@ func (m *Manager) getFormsVersion() string {
 			}
 			continue
 		}
-		defer f.Close()
 		// found and opened the version file
 		verFile = f
+		f.Close()
```

Comment:
Yeah, reading this method more closely, that was wrong. I moved the call to `defer Close` so that it's not in the `for` loop to make the linter happy.